docs/reference/images.md

```diff
@@ -313,7 +314,6 @@ $ sudo ig image inspect ghcr.io/inspektor-gadget/gadget/trace_tcp
 
 # List possible extra-info
 $ sudo ig image inspect fsnotify:main |jq keys
-WARN[0001] image signature verification is disabled due to using corresponding option 
```

Comment:
A few lines below, the doc says:
```
  "ebpf.mermaid.flowchart",
  "ebpf.mermaid.sequence",
```
But these have been renamed. The doc should be updated accordingly.